Dntc.Common/Conversion/PlannedFileConverter.cs

```diff
@@ -107,18 +112,31 @@ private IReadOnlyList<CStatementSet> GetMethodStatements(
         
         var statements = new List<CStatementSet>();
         
+        var methodInstruction = dotNetDefinedMethod.Definition.Body.Instructions.FirstOrDefault();
+
+        if (methodInstruction != null)
```

Comment:
Can this actually be null?  Won't the foreach on 140 crash if that's the case?

Reply:
fixed